kie-server-parent/kie-server-controller/kie-server-controller-websocket/src/main/java/org/kie/server/controller/websocket/client/WebSocketKieServerClient.java

```diff
@@ -277,15 +286,30 @@ public QueryDefinition getQuery(String queryName) {
                 public List<ProcessInstance> findProcessInstancesByCorrelationKey(CorrelationKey correlationKey, Integer page, Integer pageSize) {
                     return throwUnsupportedException();
                 }
-                
+
                 @Override
                 public List<ProcessInstance> findProcessInstancesByContainerId(String containerId, List<Integer> status, Integer page, Integer pageSize, String sort, boolean sortOrder) {
-                    return throwUnsupportedException();
+                    ProcessInstanceList result = null;
+                    CommandScript script = new CommandScript(Collections.singletonList((KieServerCommand) new DescriptorCommand("QueryService", "getProcessInstancesByDeploymentId", new Object[]{containerId, safeList(status), page, pageSize, sort, sortOrder})));
+                    ServiceResponse<ProcessInstanceList> listResponse = (ServiceResponse<ProcessInstanceList>) sendCommandToAllSessions(script, new WebSocketServiceResponse(true, (message) -> {
```

Comment:
use `sendCommand`instead as this is a query and not a mutating call.

Reply:
Done